universal-application-tool-0.0.1/app/controllers/admin/AdminApplicationController.java

```diff
@@ -37,6 +39,7 @@
   private final ExporterService exporterService;
   private final ProfileUtils profileUtils;
   private final Clock clock;
+  private static final int PAGE_SIZE = 10;
```

Comment:
Nit. Do we have a constants class anywhere?

Reply:
We have a handful but no generic ones I think - but it's a constant specific to the class that might want to be tweaked independently, so I would probably leave it here even so.